tests/data/remove_except_parens.py

```diff
@@ -0,0 +1,40 @@
+# These brackets are redundant, therefore remove.
+try:
+    a.something
+except (AttributeError) as err:
```

Comment:
Can you add a test case where the line is too long, like:

```
except (
    some.really.long.module.over89.chars.Error
) as really_long_name:
```

I worry that we'll run into instability issues in cases like this.

Reply:
I've added these tests and all works as expected 😄